### PR TITLE
Add cost for direction

### PR DESCRIPTION
**Pull request by kaaatsu32329**

### CODE REVIEW

**Comment by kaaatsu32329** [comment]
Screenshot from 2023-08-02 19-15-48

File: openrr-nav-viewer/examples/dwa_gui.rs [hunk 1]
```diff
@@ -46,16 +46,36 @@ fn linear_interpolate_path(path: Vec<Vec<f64>>, extend_length: f64) -> Vec<Vec<f
...
                     p0[0] + unit_diff_x * j as f64,
                     p0[1] + unit_diff_y * j as f64,
+                    direction,
                 ]);
             }
         } else {
-            interpolated_path.push(p0.to_owned());
+            interpolated_path.push({
+                let mut p = p0.to_owned();
+                p.push(0.);
```
Comment (OTL) [inline openrr-nav-viewer/examples/dwa_gui.rs:55]: I think we can use the same direction with L40.

**Review by OTL — APPROVED** [review]
あとで解説お願いします。

File: openrr-nav/examples/dwa.rs [hunk 2]
```diff
@@ -68,6 +68,8 @@ fn main() {
     maps.insert(GOAL_DISTANCE_MAP_NAME.to_owned(), goal_distance_map);
     maps.insert(OBSTACLE_DISTANCE_MAP_NAME.to_owned(), obstacle_distance_map);
     let layered = LayeredGridMap::new(maps);
+    let angles = HashMap::new();
+    // TODO: Add angles
```
Comment (OTL) [inline openrr-nav/examples/dwa.rs:72]: I can't understand what TODO is from this line.
If you need some time to implement the TODO, please add more description to explain the task more clearly.

**Review by OTL — APPROVED** [review]

File: openrr-nav/examples/dwa.rs (hunk 2 above)
Comment (kaaatsu32329) [reply openrr-nav/examples/dwa.rs]: A memo was left behind.

**Pull request merged**